src/components/AddCommentForm.js

```diff
@@ -0,0 +1,37 @@
+import { useState, useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
+import { addCommentToArticleByID } from "../utils/api";
+
+const AddCommentForm = ({ article_id, setVisibleComments }) => {
+  const [commentText, setCommentText] = useState("");
+  const { signedInUser } = useContext(UserContext);
+
+  console.log(commentText);
+
+  const handleInput = (event) => {
+    setCommentText(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (signedInUser === "") return;
+    addCommentToArticleByID(article_id, signedInUser, commentText).then(

```

Comment:
it will be there on refresh, it's just that the comments aren't ordered (however they come from api). You'd need to expand the comments and will be the last one.

Reply:
@jaykeHarrison I dont know if im doing something wrong, but my comment still isnt there, even in the view all comments. 
I refresh > view more comment - its not there.

Reply:
It's not consistent, thats the problem. I'll investigate

Reply:
<img width="743" alt="Screenshot 2022-06-16 at 16 41 22" src="https://user-images.githubusercontent.com/76402245/174109931-87513dbe-0fdc-4eb4-bbfa-646efe975270.png">

After refresh

<img width="798" alt="Screenshot 2022-06-16 at 16 41 40" src="https://user-images.githubusercontent.com/76402245/174110062-f95d57d4-b0e0-4af8-a9ab-ad751d875150.png">

Reply:
And thats all the comments, after clicking the button to view all

Reply:
ignore me, it is there. 

i'd make the view more comments button more obvious, that you have to click it again. 

otherwise, idiots like me dont know how to use it.

Reply:
Or even better, if you add a window scroll listener, it can load the next ones when you get to a certain point in the page